JS/models/militaryType.js

```diff
@@ -1,7 +1,7 @@
 const MilitaryType = {
-    TYPE_FIGHTER: 'Fighter',
-    TYPE_BOMBER: 'Bomber',
-    TYPE_TRANSPORT: 'Transport'
+  FIGHTER: "Fighter",
+  BOMBER: "Bomber",
+  TRANSPORT: "Transport",
 };
 
-module.exports =  MilitaryType;
+module.exports = MilitaryType;
```

Comment:
The best practice is to use the following syntax while exporting variables from the file:
`module.exports = { MilitaryType }`

Reply:
I've never seen something like this anywhere else. But OK :)